app/cypress/fixtures/documentTests/minimalGenericCSAFDoc.js

```diff
@@ -0,0 +1,9 @@
+import minimalDoc from './shared/minimalDoc'
+
+export default {
+  ...minimalDoc,
+  document: {
+    ...minimalDoc.document,
+    category: 'csaf_base',
+  },
+}
```

Comment:
That file should be named `app/cypress/fixtures/documentTests/minimalCSAFBaseDoc.js`

Reply:
Yes, thank you! This conflicted when rebasing on the main.

Reply:
Done in [3e5687f](https://github.com/secvisogram/secvisogram/pull/241/commits/3e5687f006764a7938f8d747c8d860d47e732920).